core/templates/dev/head/css/oppia.css

```diff
@@ -2116,27 +2116,21 @@ pre.oppia-pre-wrapped-text {
     the same line-height, margin-top and margin-bottom attributes)
 */
 .oppia-state-content-display-html > p,
-.oppia-feedback-display-html > p,
-.oppia-hint-display-html > p,
-.oppia-solution-display-html > p,
+.rte-editor > p,
```

Comment:
Can you prefix it with 'oppia-' ? This helps distinguish between css selectors that we create and those from third party libraries.

Reply:
I'll change the prefixes and also include the other selector. I didn't include it initially because I only thought to change the ones that had been previously edited, but I'll fix that now.